src/commerce-handler.js

```diff
@@ -61,6 +116,182 @@ CommerceHandler.prototype.logCommerceEvent = function(event) {
             21: ProductRemoveFromWishlist,
             22: ProductImpression
         */
+    let events = [];
```

Comment:
Use `var` instead of `const` and `let` for now. We don't have a transpiler in our kits so we sadly can't trust that some browsers will support "modern" JavaScript.